backend/package.json

```diff
@@ -23,6 +23,7 @@
     "convict": "6.2.4",
     "dotenv": "16.3.1",
     "fastify": "4.21.0",
+    "jsonwebtoken": "9.0.1",
```

Comment:
On your ticket you have a link for a library that should be used for JWT token generation

![image](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/assets/19575839/c182cdd2-29cf-45f2-baee-df69c2c72f22)

https://www.npmjs.com/package/jose

So as of now you should replace jsonwebtoken with jose
```suggestion

```